albums/Quite Good Rocket Launcher Music/2. Back In Biznitch.txt

```diff
@@ -35,13 +35,13 @@ don't fuck with us or we'll fucking murk you
 
 Mos Prob gets to show off just bezerk
 you underrated us and now here's your just deserts
-[??] (we'll|will) fucking hurt you in your inner ear
+who will fucking hurt you in your inner ear
```

Comment:
in this context, this'd be imbuing 'your just deserts' with personification; it sounds like it might be what's said, but feels like it almost certainly isn't what was written